alphastats/gui/pages/06_LLM.py

```diff
@@ -114,42 +114,91 @@ def llm_config():
         st.text("No genes of interest found.")
         st.stop()
 
+    # Separate upregulated and downregulated genes
     upregulated_genes = [
         key for key in regulated_genes_dict if regulated_genes_dict[key] == "up"
     ]
     downregulated_genes = [
         key for key in regulated_genes_dict if regulated_genes_dict[key] == "down"
     ]
 
+    # Create dataframes with checkboxes for selection
+    upregulated_genes_df = pd.DataFrame(
+        {"Protein": upregulated_genes, "Selected": [True] * len(upregulated_genes)}
```

Comment:
This displays a different type of if than the volcano and the rest of the page 
![image](https://github.com/user-attachments/assets/5b560b28-cb2d-4a96-a772-18b881fd8a2e)

check out `st.session_state[StateKeys.DATASET]._feature_to_repr_map`

Reply:
what's the consensus? do we use genes or proteins everywhere?

Reply:
`DataSet._create_id_dicts` defines the current consensus on the representation

Reply:
thanks!